src/main/java/org/frc5687/powerup/robot/Constants.java

```diff
@@ -15,6 +15,103 @@
         public static final double OUTTAKE_SPEED = -0.75;
     }
 
+    public class Auto {
+        public static final double MIN_IMU_ANGLE = -180;
+        public static final double MAX_IMU_ANGLE = 180;
+
+        public class Align {
+
+            public static final double SPEED = 0.6;
+
+            public static final double kP = 0.04;
+            public static final double kI = 0.006;
+            public static final double kD = 0.1;
+            public static final double TOLERANCE = .5;
+            public static final double MAX_OUTPUT = 0;
+            /*
+             *time the angle must be on target for to be considered steady
+             */
+            public static final double STEADY_TIME = 100;
+
+        }
+
+        public class Drive {
+
+            public static final double SPEED = 1.0;
+
+            public static final long STEADY_TIME = 100;
+            public static final long ALIGN_STEADY_TIME = 100;
+
+            public class IRPID {
+                public static final double kP = 0.05;
+                public static final double kI = 0.00;
+                public static final double kD = 0.03;
+                public static final double TOLERANCE = .5;
+
+                /**
+                 * a in the voltage-to-distance equation distance = a * voltage ^ b
+                 */
+                public static final double TRANSFORM_COEFFICIENT = 27.385;
+                /**
+                 * b in the voltage-to-distance equation distance = a * voltage ^ b
+                 */
+                public static final double TRANSFORM_POWER = -1.203;
+            }
+
+            public class EncoderPID {
+                public static final double kP = 0.05;
+                public static final double kI = 0;
+                public static final double kD = .02;
+                public static final double TOLERANCE = 1;
+            }
+
+            public class AnglePID {
+                public static final double kP = 0.04;
+                public static final double kI = 0.006;
+                public static final double kD = 0.09;
+
+                public static final double MAX_DIFFERENCE = 0.2;
+                public static final double TOLERANCE = .5;
+            }
+
+        }
+
+    }
+
+    public class Encoders {
+
+        public class Defaults {
```

Comment:
I'm not a fan of a Defaults subclass. The properties of the class are only applicable to the RightDrive and the LeftDrive, so it seems like overkill to have defaults if we're only applying them to two subclasses.